packages/zoltan2/test/color/coloring1.cpp

```diff
@@ -105,6 +111,43 @@ int validateColoring(RCP<SparseMatrix> A, int *color)
   return nconflicts;
 }
 
+int validateDistributedColoring(RCP<SparseMatrix> A, int *color, int rank){
```

Comment:
The input `A` can never be null, so just pass in a reference (`const SparseMatrix&`) instead of an RCP.

Reply:
Done.